types/key/hardware_attestation.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package key
+
+import (
+	"crypto"
+	"encoding/json"
+	"fmt"
+)
+
+var ErrUnsupported = fmt.Errorf("hardware attestation key not supported on this platform")
```

Comment:
nit: since the error var is generic for this package
```suggestion
var ErrUnsupported = fmt.Errorf("key type not supported on this platform")
```